tests/bindings-test2/dotnet/shared.csproj

```diff
@@ -7,10 +7,18 @@
     <LangVersion>latest</LangVersion>
     <IsBindingProject>true</IsBindingProject>
 
+    <RootTestsDirectory>$([System.IO.Path]::GetFullPath('$(MSBuildProjectDirectory)\..\..\..'))</RootTestsDirectory>
     <BindingsTest2Directory>$(RootTestsDirectory)\bindings-test2</BindingsTest2Directory>
     <TestLibrariesDirectory>$(RootTestsDirectory)\test-libraries</TestLibrariesDirectory>
   </PropertyGroup>
 
+  <Import Project="$(RootTestsDirectory)/common/shared-dotnet.csproj" />
+
+  <ItemGroup>
+    <PackageReference Include="NUnitLite" Version="3.12.0" />
+    <ProjectReference Include="$(RootTestsDirectory)\bindings-test\dotnet\$(_PlatformName)\bindings-test.csproj" />
```

Comment:
The rest of the file changes are / but here you have \ in the path, just a nit if it matters at all

Reply:
It doesn't matter, both \ and / work on macOS.